src/js/screen_gfx.js

```diff
@@ -2,9 +2,14 @@
 
 import rgba from "color-rgba";
 
-const fontFamily = '"sans-serif-condensed", "Arial Narrow", sans-serif';
-export const shieldFont = (size) => `bold ${size}px ${fontFamily}`;
-export const fontSizeThreshold = 12;
+const shieldFontFamily =
+  'Overpass, "sans-serif-condensed", "Arial Narrow", sans-serif';
```

Comment:
While we’re at it loading a Web font from Google Fonts, I’m partial to either Roboto Condensed or Ubuntu Condensed. It already looks pretty good (and indistinguishable from the previous Arial Narrow) even before the various dimensional tweaks in the shield definitions.

![Roboto Condensed](https://user-images.githubusercontent.com/1231218/227705730-dff16c74-072f-497c-a491-a19568bdfd94.png)